docs/website/content/docs/v0.1/Getting Started/concepts.md

```diff
@@ -12,17 +12,25 @@ The Metal Controller Manager provides a few custom resources (CRDs) in the manag
 These define a desired deployment environment for Talos.
 This includes things like which kernel to use, kernel args to pass, and the initrd to use.

```

Comment:
"Sidero allows you to define a default environment, but also to override the environment at the ServerClass or Server level, if you have requirements for different kernels or kernel parameters for some nodes."